source/23_visual/index.rst

```diff
@@ -1,4 +1,4 @@
-Graphics and Visualization
+Diagrams and Visualization
```

Comment:
Can not find images in this page.

Reply:
The images can be automatically generated due to `diagrams.mk`